report/sections/secondary-groups.tex

```diff
@@ -1,3 +1,25 @@
 \section{Secondary Groups}
 
-\todo{TODO: Explain format read from the Computer Vision group (and results if we manage to get some examples of their work), and output given to the Emotion Synthesis group}
+%TODO: Explain format read from the Computer Vision group (and results if we manage to get some examples of their work), and output given to the Emotion Synthesis group
+Our Secondary groups were Computer vision and Emotion Synthesis. Our task in final pipeline was to read the input given by CV group and send the output to ES group.
+
+\subsection{Computer Vision Group}
+%Format read from CV Group
+After training our models with the Bosphorus database we need to predict an emotion by reading input given from CV group. They are giving input in numpy array containing $24$ landmarks. We did dimensionality reduction while training our model so we are using the same pca model to do the equal amount of dimensionality reduction for the given input. 
```

Comment:
- [ ] 'from **the** CV group'
- [ ] 'input in **a** numpy array'
- [ ] pca -> PCA
- [ ] Rephrase final sentence it's a bit of a run-on